### PR TITLE
`<SelectableAccordion />` -  add disabled property to items list

### PR DESCRIPTION
**Pull request by domakas**
### 🔦 Summary

Added disabled state for <SelectableAccordion />. When providing this property, controls becomes disabled, and all interactions are turned off.

### ✅ Checklist

- [x] 👨‍💻 API change is approved by the UI Infra Developers @milkyfruit @mykas elevant team member -->
- 📚 Change is documented
  - [x] Story
  - [x] API description
- 🔬 Change is tested
  - [x] Component

### CODE REVIEW

**Comment by WixBuildServer** [comment]
### 🤖 Hi, I'm #Janet!

  A deploy preview of your PR was successfully released 🖼.

  Project Name: **wix-ui-tpa**

  You can find it under one of the following URLs:
  * https://bo.wix.com/pages/wix-ui-tpa.pr_6928/

**Comment by WixBuildServer** [comment]
### 🤖 Hi, I'm #Janet!

  A deploy preview of your PR was successfully released 🖼.

  Project Name: **wix-style-react**

  You can find it under one of the following URLs:
  * https://preview.wix-style-react.com/pr_6928

**Review by milkyfruit — CHANGES_REQUESTED** [review]
@domakas please change disabled from being for all items to disabled per item. Then it will be consistent with `<Accordion/>` component:
https://www.wix-style-react.com/?activeTab=Description&path=%2Fstory%2Fcomponents-api-components--accordion
image

**Review by milkyfruit — APPROVED** [review]
@domakas looks great now!
@mykas could you check the code and merge the PR?

File: packages/wix-style-react/src/SelectableAccordion/Item/Item.js [hunk 1]
```diff
@@ -131,18 +151,20 @@ export default class SelectableAccordionItem extends React.PureComponent {
...
-    const { open, verticalPadding } = this.props;
+    const { open, verticalPadding, disabled } = this.props;
 
     return (
       <WixStyleReactContext.Consumer>
         {({ reducedSpacingAndImprovedLayout }) => (
           <div
             data-hook={dataHooks.item}
             data-state={open ? 'open' : 'collapsed'}
+            data-disabled={disabled ? 'disabled' : ''}
```
Comment (mykas) [inline packages/wix-style-react/src/SelectableAccordion/Item/Item.js:162]:
```suggestion
            data-disabled={disabled}
```

File: packages/wix-style-react/src/SelectableAccordion/SelectableAccordion.uni.driver.js [hunk 2]
```diff
@@ -31,5 +31,17 @@ export const selectableAccordionDriverFactory = (base, body) => {
...
+    /**
+     * Checks whether the item is disabled by item index
+     * @param {number} idx Item index
+     * @returns {Promise<boolean>}
+     */
+    isItemDisabledAt: async idx => {
+      const item = await getItemAt(base, idx);
+      const itemState = await getDataAttributeValue(item, dataAttr.DISABLED);
+
+      return itemState === 'disabled';
```
Comment (mykas) [inline packages/wix-style-react/src/SelectableAccordion/SelectableAccordion.uni.driver.js:44]: Disabled property for data-disable should be boolean instead

**Review by mykas — APPROVED** [review]
Good job! lets wait for build to get green and then I'll merge it.

**Comment by WixBuildServer** [comment]
### 🤖 Hi, I'm #Janet!

A deploy preview of your PR was successfully released 🖼.

Project Name: **wix-ui-tpa**

You can find it under one of the following URLs:
* https://bo.wix.com/pages/wix-ui-tpa.pr_6928/

**Comment by WixBuildServer** [comment]
### 🤖 Hi, I'm #Janet!

A deploy preview of your PR was successfully released 🖼.

Project Name: **wix-style-react**

You can find it under one of the following URLs:
* https://preview.wix-style-react.com/pr_6928

**Pull request merged**